jscomp/others/belt_List.ml

```diff
@@ -692,6 +692,16 @@ let setAssocU xs x k eq =
 
 let setAssoc xs x k eq = setAssocU xs x k (fun [@bs] a b -> eq a b)
 
+let sort xs cmp =
+  let arr = toArray xs in
+  Belt_SortArray.stableSortInPlaceBy arr cmp;
+  fromArray arr
+
+let sortU xs cmp =
+  let arr = toArray xs in
+  Belt_SortArray.stableSortInPlaceByU arr cmp;
+  fromArray arr
+
```

Comment:
In general, in belt, we should define `sortU` first and wrap `sort` like this

```ocaml
let sort xs cmp = sortU xs (fun[@bs] x y -> cmp x y)
```
The thing is in your `sort`, you are calling `stableSortInPlaceBy` instead of `stableSortInPlaceByU` which is slower

Reply:
Fixed. Thanks!